lib/foreman_maintain/cli/base.rb

```diff
@@ -49,6 +50,7 @@ def self.tags_option
         option '--tags', 'tags',
                'Limit only for specific set of labels. ' \
                  '(Use list-tags command to see available tags)' do |tags|
+          raise ArgumentError, 'value not specified' if tags.nil? || tags.empty?
```

Comment:
Sry, merged too fast. Good call: added tests in separate PR https://github.com/iNecas/foreman_maintain/pull/28